webrender/src/mask_cache.rs

```diff
@@ -44,6 +44,14 @@ impl ClipRegion {
         }
     }
 
+    pub fn for_clip_node_with_local_clip(local_clip: &LocalClip) -> ClipRegion {
```

Comment:
Okay. I changed this to create_for_clip_node_with_local_clip. I'll open a PR to make the other names coherent with it as well.